pytest_html/extras.py

```diff
@@ -8,17 +8,33 @@
 FORMAT_TEXT = 'text'
 FORMAT_URL = 'url'
 
+TYPE_JPG = ('image/jpeg', 'jpg')
+TYPE_PNG = ('image/png', 'png')
+TYPE_SVG = ('image/svg+xml', 'svg')
 
-def extra(content, format, name=None):
-    return {'name': name, 'format': format, 'content': content}
+
+def extra(content, format, name=None, type=None):
```

Comment:
I think it would make sense to add `mime_type` and `extension` as optional arguments. Note that `type` is reserved.